installer/templates/phx_assets/app.css

```diff
@@ -0,0 +1,52 @@
+/* This file is for your main application css. */
+@import "./phoenix.css";<%= if live do %>
+@import "../node_modules/nprogress/nprogress.css";
+
+/* LiveView specific classes for your customizations */
```

Comment:
I think this should go in phoenix.css, everything else we use in scaffold or home pages go there.